src/app/keyring/components/AdvKeyGenOptions.js

```diff
@@ -22,7 +23,11 @@ l10n.register([
 
 export default function AdvKeyGenOptions({value: {keyAlgo, keySize, keyExpirationTime}, onChange, disabled}) {
   const context = React.useContext(KeyringOptions);
-  const handleDateChange = moment => onChange({target: {id: 'keyExpirationTime', value: moment}});
+  const tomorrow = addDays(new Date(), 1);
+  const farFuture = addYears(tomorrow, 80);
+  // TODO #moment-to-data-fns-migration
```

Comment:
I'm not a fan of TODO comments. Pros and cons for it, but we currently don't have them in the code base. I would not like to introduce them to track tech-dept.

Reply:
Shall we keep "tags"?

Reply:
I just think I don't want the effort to keep these tags up-to-date. Real world example: imagine the migration to date-fns is delayed, next time we use `moment` we need to add the tag to mark the line. Over time this does not work 100%. So what happens is, if I finally do the migration to date-fns it is not enough to search for the tag and change these places, I also have to take into account that there might be other places where `moment` is in use.

Reply:
I don't think we **need** to add the tag. I also don't think this should be consistent over the codebase. There was a line (where `toDate()` is called) which doesn't have the `moment` keyword in it but has to be changed during migration. And this might help to quicker identify it. 
Of course I'm not strong on it, it is opinionated practice

Reply:
Ah ok, I didn't see this meaning behind the tag. Quite cryptic. Think I prefer not to have this mental load while reading through the code.

Reply:
Removed